src/inspect_ai/_view/www/src/components/ErrorPanel.mjs

```diff
@@ -43,11 +43,14 @@ export const ErrorPanel = ({ id, classes, title, error }) => {
       >
         <div>
           Error: ${message || ""}
-          <pre style=${{ fontSize: "0.8rem" }}>
+          ${stack &&
```

Comment:
I'm not totally sure that this is necessary (but I may not understand the change). The stack is being displayed below the error currently - does that make this change unnecessary?

Reply:
Yeah, so this is more because I wanted to use the ErrorPanel in a slightly different way which is maybe incorrect. I just wanted to throw an error but didn't care about the stack trace - in my case it was showing "Unauthorised", so I wanted a relevant "error panel" (in particular reuse the styles etc) but there was no relevant stack trace.

Happy to find another way around that if you want, but that was the underlying reason

Reply:
this makes sense!